src/run.ts

```diff
@@ -0,0 +1,87 @@
+import { Argv } from 'yargs';
+import { Command, Helper, OptionsHelper } from '@dojo/cli/interfaces';
+import { join, relative, resolve } from 'path';
+import * as chalk from 'chalk';
+import createDir from '@dojo/cli-create-app/createDir';
+import renderFiles from '@dojo/cli-create-app/renderFiles';
+
+import register from './register';
+import run from './run';
+
+const pkgDir: any = require('pkg-dir');
+const packagePath = pkgDir.sync(__dirname);
+
+/**
+ * @type CreateWidgetArgs
+ *
+ * Arguments that can be passed when creating a widget using the Dojo CLI
+ *
+ * @property component       Generate a Custom Element wrapper
+ * @property name            Name used for the generated widget
+ * @property styles          Path to place generated CSS files
+ * @property tests           Path to place generated test files
+ */
+export interface CreateWidgetArgs extends Argv {
+	component: boolean;
+	name: string;
+	styles: string;
+	tests: string;
+}
+
+function getDirectoryNames(args: CreateWidgetArgs, folderName: string) {
+	const dirs = [ folderName ];
+
+	!args.styles && dirs.push(`${folderName}/styles`);
```

Comment:
feels like it would be clearer if this was 
```ts
if (args.styles === undefined) {

}
```

or just `if (!args.styles)` even